frontend/src/components/GSelectCloudProfile.vue

```diff
@@ -85,15 +93,15 @@ const selectItems = computed(() => {
     )
   }
 
-  if (cloudProfileList.value.length > 0) {
+  if (cloudProfilesRef.value.length > 0) {
     items.push(
-      ...cloudProfileList.value.map(profile => {
-        const cloudProfileRef = {
+      ...cloudProfilesRef.value.map(profile => {
+        const cloudProfileRefListEntry = {
           name: profile.metadata.name,
           kind: 'CloudProfile',
         }
         return {
-          value: cloudProfileRef,
+          value: cloudProfileRefListEntry,
```

Comment:
```suggestion
        const cloudProfileRef = {
         name: profile.metadata.name,
         kind: 'CloudProfile',
       }
       return {
          value: cloudProfileRef,
```
keep `cloudProfileRef` naming